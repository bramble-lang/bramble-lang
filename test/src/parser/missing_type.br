fn my_main() {

    
    let x := 1;
    return;
}