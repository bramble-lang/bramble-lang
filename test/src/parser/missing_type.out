Error: Parser: L4: Expected type specification after x
